### PR TITLE
Feature/save button

### PR DESCRIPTION
**Pull request by nkolarec**
I need a code check before merging and closing issue #8MapViewActivity  just in case because I was adding a DialogFragment. I'm going to work on rest before doing any saving functions in Android.

### CODE REVIEW

File: LocationTrack-app/app/src/main/java/hr/fer/ruazosa/locationtrack/MapViewActivity.kt [hunk 1]
```diff
@@ -66,7 +76,18 @@ class MapViewActivity : AppCompatActivity(), OnMapReadyCallback {
         mMap = googleMap
     }
 
+    private fun openDialog(){
+        val dialog = SaveDialog()
+        dialog.show(supportFragmentManager, "SaveDialog")
+    }
 
+    private fun getLatitude(): Double? {
```
Comment (MrPranklin) [inline LocationTrack-app/app/src/main/java/hr/fer/ruazosa/locationtrack/MapViewActivity.kt]: privatni getteri i setteri nemaju bas smisla

Comment (MrPranklin) [inline LocationTrack-app/app/src/main/java/hr/fer/ruazosa/locationtrack/MapViewActivity.kt]: privatni getteri i setteri nemaju bas smisla

File: LocationTrack-app/app/src/main/res/layout/dialog_layout.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (MrPranklin) [inline LocationTrack-app/app/src/main/res/layout/dialog_layout.xml]: mozda dodati i neka dva textviewa u kojima pise lat i long, cisto da bude malo veci dialog

File: LocationTrack-app/app/src/main/java/hr/fer/ruazosa/locationtrack/MapViewActivity.kt (hunk 1 above)
Comment (nkolarec) [reply LocationTrack-app/app/src/main/java/hr/fer/ruazosa/locationtrack/MapViewActivity.kt]: ah, vidim. makla sam ih pa ako bu trebalo onda cemo poslije ih dodati po potrebi.

File: LocationTrack-app/app/src/main/res/layout/dialog_layout.xml (hunk 2 above)
Comment (nkolarec) [reply LocationTrack-app/app/src/main/res/layout/dialog_layout.xml]: Evo dodala sam ih, sad je ipak veci taj dialog.

**Pull request merged**